### ISSUE TITLE
Feature: Table of uniqnames to reference from attendance

### ISSUE THREAD

**Issue by bharadwaj-a** [FAMNM/famnm-website-backend#6]
We could use this table to confirm when new uniqnames are added and also make sure that typos are noticed when inserting into attendance. 

In future, we could even look at autocompleting with this list.

**Comment by AmandaKline** [comment]
Isn't this duplicate data we have stored already in other systems? Definitely might be useful, but I'm not sure it's necessary for an MVP.

**Comment by bharadwaj-a** [comment]
Yeah but I'm thinking towards how meetings would be run. This is to address the concern that we might enter a uniqname incorrectly during a meeting. MVP, this isn't necessary but I'm not sure we want to rollout the MVP for use without some kind of way of making sure that the uniqnames are actually for the people in the meeting.
